allennlp/data/dataset_readers/stanford_sentiment_tree_bank.py

```diff
@@ -0,0 +1,125 @@
+from typing import Dict, List
+import logging
+
+from overrides import overrides
+from nltk.tree import Tree
+import tqdm
+
+from allennlp.common import Params
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import LabelField, TextField, Field
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers import TokenIndexer, SingleIdTokenIndexer
+from allennlp.data.tokenizers import Token
+from allennlp.common.checks import ConfigurationError
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@DatasetReader.register("sst_tokens")
+class StanfordSentimentTreeBankTokensDatasetReader(DatasetReader):
+    """
+    Reads tokens and their sentiment labels from the Stanford Sentiment Treebank.
+
+    Parameters
+    ----------
+    token_indexers : ``Dict[str, TokenIndexer]``, optional (default=``{"tokens": SingleIdTokenIndexer()}``)
```

Comment:
I don't think that these backticks inside of the default block actually render correctly.  Not a huge deal, just FYI - it's probably not worth worrying about adding them, because they don't work, anyway.  I've also taken to putting more complex default values like this in the text description, where they will actually get rendered correctly.